tools/compile_shader.sh

```diff
@@ -0,0 +1,54 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         ::::::::             #
+#    compile_shader.sh                                  :+:    :+:             #
+#                                                      +:+                     #
+#    By: fbes <fbes@student.codam.nl>                 +#+                      #
+#                                                    +#+                       #
+#    Created: 2022/03/03 02:38:19 by fbes          #+#    #+#                  #
+#    Updated: 2022/03/03 04:05:03 by fbes          ########   odam.nl          #
+#                                                                              #
+# **************************************************************************** #
+
+#!/bin/bash
+
+if [ "$#" -ne 1 ]; then
+	exit 1
+fi
+
+SHADERTYPE="${1##*.}"
+
+echo "/* ************************************************************************** */"
+echo "/*                                                                            */"
+echo "/*                                                        ::::::::            */"
+echo "/*   lol.c                                              :+:    :+:            */"
+echo "/*                                                     +:+                    */"
+echo "/*   By: W2wizard <lde-la-h@student.codam.nl>         +#+                     */"
+echo "/*                                                   +#+                      */"
+echo "/*   Created: 2022/02/17 22:34:59 by W2wizard      #+#    #+#                 */"
+echo "/*   Updated: 2022/02/17 22:34:59 by W2wizard      ########   odam.nl         */"
+echo "/*                                                                            */"
+echo "/* ************************************************************************** */"
+echo ""
+echo "// If you wish to modify this file edit the .vert or .frag file!"
+echo ""
+echo "#include \"MLX42/MLX42_Int.h\""
+echo ""
+echo "const char	*g_${SHADERTYPE}_shader = \"$(head -n 2 $1 | sed -n 2p | tr -d '\r')\\n\""
```

Comment:
This should be more like `const char* ${SHADERTYPE}_shader` since we don't follow the norme anymore.

Reply:
Feel free to modify the pull request, you should have write access to my branch